packaging/nuget/nservicebus.testing.fakes.nuspec

```diff
@@ -20,7 +20,7 @@
   <files>
     <file
       src="..\..\src\NServiceBus.Testing.Fakes\*.cs"
-      target="content\Testing\Fakes"
+      target="content\App_Packages\NSB.Testing.Fakes.$version$"
```

Comment:
The folder name should match the id of the package, so we should either drop the "Testing" here, or change the id of the package to be `NServiceBus.Testing.Fakes.Sources`.